admin_manual/configuration_files/external_storage/local.rst

```diff
@@ -16,7 +16,7 @@ are on Ubuntu Linux::
  sudo -u www-data chmod -R 0750 /localdir
  
 See :ref:`label-setting-strong-perms` for information on finding your HTTP 
-user with ``phpinfo``. 
+user with :ref:`label-phpinfo`. 
  
```

Comment:
If you wouldn't mind, it would be good to backport the other pages as they are the same in 8.2 and 8.1.